src/_guides/language/language-tour.md

```diff
@@ -2326,6 +2318,18 @@ is true, the assertion succeeds and execution
 continues. If it's false, the assertion fails and an exception (an
 [AssertionError][]) is thrown.
 
+When exactly do assertions work?
+That depends on the tools and framework you're using:
+
+* Flutter enables assertions in [debug mode.][Flutter debug mode]
+* Development-only tools such as [dartdevc][]
+  typically support assertions by default.
```

Comment:
I can't seem to find what `webdev` uses when it runs the dev compiler, but  I think changing to "typically enable assertions by default" might be more accurate here